rbac/api/cross_access/view.py

```diff
@@ -241,7 +241,7 @@ def update_status(self, car, status):
             self._with_dual_write_handler(
                 car,
                 ReplicationEventType.APPROVE_CROSS_ACCOUNT_REQUEST,
-                lambda dual_write_handler, cross_account_roles: dual_write_handler.generate_relations_to_add_roles(
+                lambda dual_write_handler, cross_account_roles: dual_write_handler.generate_relations_reset_roles(
```

Comment:
is this intent to switch from `generate_relations_to_add_roles` to `generate_relations_reset_roles` ?

Reply:
Good question, after thinking about it, should use the generate_relations_to_add_roles to reduce the interruption. Updated